java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/instance/SystemTestDeploymentContext.java

```diff
@@ -207,12 +208,12 @@ private void uploadJarsAndDockerImages() throws IOException, InterruptedExceptio
         }
         UploadDockerImages.builder()
                 .baseDockerDirectory(parameters.getDockerDirectory())
-                .ecrPrefix(parameters.getSystemTestShortId())
-                .account(parameters.getAccount())
-                .region(parameters.getRegion())
-                .stacks(List.of("SystemTestStack"))
                 .ecrClient(EcrRepositoryCreator.withEcrClient(ecr))
-                .build().upload(ClientUtils::runCommandLogOutput);
+                .build().upload(ClientUtils::runCommandLogOutput, StacksForDockerUpload.builder()
+                        .ecrPrefix(parameters.getSystemTestShortId())
+                        .account(parameters.getAccount())
+                        .region(parameters.getRegion())
+                        .stacks(List.of("SystemTestStack")).build());
```

Comment:
The version was previously defaulted to `SleeperVersion.getVersion()` in the builder, and it still needs to be set here.